tasks/main.yml

```diff
@@ -17,6 +17,28 @@
     - iptables-services
   become: True
 
+- name: disable selinux to permit using hosts docker for builds
```

Comment:
Should be "host's" or perhaps "host" if the single quote causes issues for e.g. syntax highlight or some such thing.

Reply:
True.